index.html

```diff
@@ -1688,40 +1698,62 @@
             </dd>
           </dl>
           <ol>
-            <li>If the <a>presentation connection state</a> of
-            <code>presentationConnection</code> is not <code>connected</code>
-            or <code>connecting</code>, then abort these steps.
+            <li>Signal the intention to close the corresponding
+            <code>PresentationConnection</code> to the <a>destination browsing
+            context</a>, passing the <code>closeReason</code> to that context.
             </li>
```

Comment:
One thing that does not strike me as clear is whether "signal" involves going through the closing handshake and thus waiting for the ack from the other side. If so, this should take place behind the scenes and I would suggest using "Start to signal", or clarifying what "signal" means somehow.

In particular, I assume that the "close the presentation connection" procedure is to run in the same run of the event loop when that procedure is called by the close() method.